### PR TITLE
barista: Fix chart behavior switch example.

### LINKED ISSUE #397: Barista: Switching metrics example on chart page broken
There seems to be an issue with the switching metrics chart example. It gets redrawn on hover and the tooltip is empty.

### PR DESCRIPTION
**Pull request by lara-aigmueller**
### <strong>Pull Request</strong>

fixes #397 

<hr>
Hi, thank you for contributing to Barista with this pull request (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please choose the type appropriate for the changes below: <br>

#### Checklist

- [ ] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts [hunk 1]
```diff
@@ -15,112 +15,110 @@
  */
 
 import { Component } from '@angular/core';
-
 import { generateData } from '../chart-data-utils';
 
+const cpuUsageSeries: Highcharts.IndividualSeriesOptions[] = [
```
Comment (thomaspink) [inline libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts]: I think we should move these array back into the class as a prop so it appears in the example source in barista.

File: libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts [hunk 2]
```diff
@@ -15,112 +15,110 @@
...
+  {
+    name: 'Number of process group instances',
+    type: 'column',
+    yAxis: 1,
+    data: generateData(40, 1, 4, 1370304000000, 900000),
+    color: '#006bba',
+  },
+];
+
+const retransmissionSeries: Highcharts.IndividualSeriesOptions[] = [
```
Comment (thomaspink) [inline libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts]: I think we should move these array back into the class as a prop so it appears in the example source in barista.

File: libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts [hunk 3]
```diff
@@ -15,112 +15,110 @@
...
+  {
+    name: 'Server usage',
+    type: 'line',
+    yAxis: 1,
+    data: generateData(40, 15, 100, 1370304000000, 900000),
+    color: '#f5d30f',
+  },
+];
+
+const connectivitySeries: Highcharts.IndividualSeriesOptions[] = [
```
Comment (thomaspink) [inline libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts]: I think we should move these array back into the class as a prop so it appears in the example source in barista.

File: libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts (hunk 1 above)
Comment (lara-aigmueller) [reply libs/examples/src/chart/chart-behavior-switch-example/chart-behavior-switch-example.ts]: Yep, also thought about that... was not sure if this is critical information for the example. But I'll move it back again! :)

**Pull request merged**